### DISCUSSION TITLE
Load of the script slow when multiples accounts

### DISCUSSION THREAD

**Discussion by Decafeinato** [InboxSDK/InboxSDK#920] [General]
When using the extension, I notice that the load of the sdk is very slow when I'm using a Chrome profile with multiple accounts connected ( 4 accounts), is it only me? I can't find guidelines issues or anything on this topic I would to have feedbacks on this.

Thanks,
David

**Comment by wegry** [comment]
Hey @Decafeinato, thanks for the question. How slow is 'very slow' in this case? And is it measurably slower than a Chrome profile without multiple accounts connected?

**Reply by Decafeinato** [reply]
Hi, It was taking 3s for 2 users, 6 for 3 users... 15s to 4 users .

But I found out that this was a side effect of a bundling mistake on my end.

Adding following code on the background script helped : 
`  
  if (msg.type === 'inboxsdk__injectPageWorld' && sender.tab) {
    if (chrome.scripting) { // MV3 
      chrome.scripting.executeScript({ target: { tabId: sender.tab.id }, world: 'MAIN', files: ['pageWorld.js'] });
      response(true);
    } else { // MV2 fallback. Tell content script it needs to figure things out. 
      response(false);
    }
  }
`
